scripts/findAndMatch.js

```diff
@@ -0,0 +1,319 @@
+//GAME - Find & Match
+
+//To be continued: Match the artist with his/her instrument
+
+var msg1 = document.getElementById("msg1");
+var msg2 = document.getElementById("msg2");
+var msg3 = document.getElementById("msg3");
+var startBtn = document.getElementById("start");
+var stopBtn = document.getElementById("stop");
+var outcome = document.getElementById("outcome");
+var p = document.createElement("h5");
+
+startBtn.onclick = startGame;
+stopBtn.onclick = stopCounter;
+
+var randomNames = [];
+var names = [
+  "Hetfield",
+  "Malmsteen",
+  "Trujillo",
+  "Hammet",
+  "Lars",
+  "Halford",
+  "Heafy",
+  "Lemmy",
+  "Ozzy"
+];
+
+var timeLeft = 5;
+var timer;
+var totalLives = 3;
+var remainingLives = 3;
+var key;
+
+function startCounter() {
+  timer = window.setInterval(counter, 1000);
+}
+
+function counter() {
+  timeLeft--;
+  if (timeLeft < 0) {
+    window.clearInterval(timer);
+    remainingLives--;
+    timeLeft = 5;
+    startCounter();
+  }
+  msg2.innerHTML = timeLeft + " sec remaining";
+  msg3.innerHTML =
+    "<hr>" +
+    remainingLives +
+    " / " +
+    totalLives +
+    " " +
+    '<i class="fas fa-heart" style = "color: indianred"></i>';
+  if (remainingLives == 0) {
+    msg1.innerHTML =
+      "GAME OVER!!!" + "<br><br>" + "You did not guess all the artists...";
+    msg1.style.cssText = "color: indianred; font-size: 28px;";
+    msg2.innerHTML = "";
+    msg3.innerHTML = "Please refresh and try again.";
+    outcome.style.cssText = "display: none;";
+    window.clearInterval(timer);
+    startBtn.disabled = true;
+    stopBtn.disabled = true;
+  }
+}
+
+function stopCounter() {
+  window.clearInterval(timer);
+  startBtn.disabled = false;
+}
+
+function startGame() {
+  timeLeft = 10;
+  remainingLives = 3;
+  randomNames = names.slice();
+  randomNames.sort(function() {
+    return 0.5 - Math.random();
+  }); //randomize an array (BEST WAY)
+  msg2.innerHTML = timeLeft + " sec remaining";
+  msg3.innerHTML +=
+    "<hr>" +
+    remainingLives +
+    " / " +
+    totalLives +
+    " " +
+    '<i class="fas fa-heart" style = "color: indianred"></i>';
+  document.getElementById("instructions").style.cssText = "display: none;";
+  msg1.style.cssText = "color: black;";
+  startBtn.disabled = true;
+  findArtist();
+  buildBoard();
+  startCounter();
+}
+
+function findArtist() {
+  key = randomNames[Math.floor(Math.random() * (randomNames.length - 1))];
+  msg1.innerHTML = "Find " + '<span id = "name">' + key + "</span>";
+}
+
+function buildBoard() {
+  for (var i = 0; i < randomNames.length; i++) {
+    outcome.innerHTML +=
+      '<p class = "box" onmouseover = "showName()" onmouseout = "hideName()" onclick = "guessName()">Hidden ' +
+      "</p>";
+  }
+}
+
+function showName() {
+  for (var i = 0; i < randomNames.length; i++) {
+    outcome.children[i].value = randomNames[i];
+  }
+  event.target.innerText = event.target.value;
+  event.target.style.cssText =

```

Comment:
Instead of modifying the CSS directly it's much better to use a class to apply styling.
```
.myClass {
   background-red;
}

event.target.classList.add('myClass');
// you can also remove classes
event.target.classList.remove('myClass');

//note that it works on any DOM element:
document.getElementById('cacat').classList.add('myClass')

```